sycl/test/basic_tests/accessor/atomic_zero_dimension_accessor.cpp

```diff
@@ -0,0 +1,66 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_fpga %s -o %t.out
```

Comment:
Can we add `-fsyntax-only` to get rid of unnecessary compilation steps?

Reply:
thanks - good call. Done.